Example/MoyaComparison/MoyaComparison/Models/MultipleJSONConformance.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright 2017 niji
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import JSONCodable
+import ObjectMapper
+import SwiftyJSON
+
+/// Swifty protocol to use Generics for parsing with SwiftyJSON
+protocol Swifty {
+  init(json: JSON)
+  func toJSON() -> JSON
+}
+
+extension Swifty {
+  public func toJSON() -> JSON {
+    return JSON(self)
+  }
+}
+
+/// MultipleJSONConformance protocol is conformance of JSONCodable, Mappable, Swifty protocols
+protocol MultipleJSONConformance: JSONCodable, Mappable, Swifty {}
```

Comment:
I'm not a fan of these for two reasons:

1. In such cases, it's better to use a `typealias MultipleJSONConformance = JSONCodable & Mappable & Swifty`. The difference is that:

  * when you have `typealias AB = A & B`, if you have `class C1: A` then `class C2: C1, B` then `C2` is considered being of typealias `AB` too (`C2 is AB` will be true), because it indeed conforms to `A` (from one side)  and `B` (from another place).
  * While if you use `protocol AB: A, B` instead, then `C2 is AB` will be false, because C2 will conform to A and will also conform to B but that doesn't make it explicitly conform to that new and different protocol called AB

2. For that particular case of your example, the name is not really explicit, and I think it would be way more understandable to explicitly use `class MyModel: JSONCodable & Mappable & Swifty` so that when you read the example you see clearly that it works for the 3 son libs, instead of wondering which are contained in `MultipleJSONCOnformance` (that's even worse with `DualJSONConformance` as I always wonder which 2 of the 3 you mean)